includes/restoreMappings.js

```diff
@@ -0,0 +1,112 @@
+// Copyright © 2017, 2023 IBM Corp. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+'use strict';
+
+const error = require('./error.js');
+const debug = require('debug');
+
+const mappingDebug = debug('couchbackup:mappings');
+
+class Restore {
+  constructor(db) {
+    this.db = db;
+    this.totalDocsRestored = 0;
+    this.batchCounter = 0;
+  }
+
+  /**
+   * Mapper for converting a backup file line to an array of documents pending restoration.
+   *
+   * @param {object} backupLine object representation of a backup file line {lineNumber: #, line: '...'}
+   * @returns {array} array of documents parsed from the line
+   */
+  backupLineToDocsArray = async(backupLine) => {
```

Comment:
I instinctively spaced it, but, but the linter moaned at me with [`space-before-function-paren`](https://eslint.org/docs/latest/rules/space-before-function-paren).

It's configurable to be different for arrow functions, but if we want to change that I suggest we do it in a follow up clean up, rather than landing in a mixed state meanwhile. Also the rule is recently deprecated and moved into a stylistc plugin, so I guess maybe we'll need to choose a style at some point if we want to keep rules like that.